Create 2024-11-14.md (#208)
department-of-veterans-affairs/developer-portal-content: content/verification/veteran-confirmation/release-notes/2024-11-14.md

> We've added a new response for Veteran Confirmation /status. Not confirmed - Requires manual review is now an option that can be returned. This is a response that is received from our backend data source and should give more details for individuals who are Not confirmed as title 38 Veterans.

Reviewer: ```suggestion
We've added a new response for Veteran Confirmation /status:
- `Not confirmed - Requires manual review` can now be returned. 

This response comes from our backend data source and should give more details about individuals who are not confirmed as Title 38 Veterans.
```
